amkotlinutil/src/main/java/dk/adaptmobile/amkotlinutil/extensions/ConductorExtensions.kt

```diff
@@ -159,3 +161,50 @@ var Controller.brightness: Float?
 
 val Router.secondLastController: Controller?
     get() = if (this.backstack.size > 1) this.backstack[this.backstack.lastIndex - 1].controller() else null
+
+
+fun Router.isNotEmpty(): Boolean {
+    return !isEmpty()
+}
+
+fun Router.isEmpty(): Boolean {
+    return backstackSize == 0
+}
+
+fun Router.goBack(amount: Int = 1, data: Any? = null) {
+    if (amount <= 1) {
+        popCurrentController()
+    } else {
+        val tempBackStack = backstack
+        for (i in 1..amount) {
+            tempBackStack.removeAt(tempBackStack.lastIndex)
+        }
```

Comment:
Where do we use `i`?

Reply:
Nowhere, is there a way to iterate over a range without it?

Reply:
Do a foreach without index ;)